### PR TITLE
#664 - Added default graph range for environment

### PR DESCRIPTION
**Pull request by KZavorotna**
Is required for https://github.com/withthegrid/platform-client/pull/746

Is related to https://github.com/withthegrid/platform-client/pull/746

### CODE REVIEW

File: src/models/environment.ts [hunk 1]
```diff
@@ -25,6 +25,7 @@ const schema = Joi.object().keys({
   locale: localeSchema.required().example('en'),
   expiresAt: Joi.date().allow(null).required().example(null),
   createdAt: Joi.date().required().example('2019-12-31T15:23Z'),
+  defaultGraphRange: Joi.string().allow(null).required().example('30d'),
```
Comment (yarsky-tgz) [inline src/models/environment.ts:28]: is `allow(null)` needed? It's not reflected on type.

**Review by yarsky-tgz — CHANGES_REQUESTED** [review]
One small comment

File: src/models/environment.ts (hunk 1 above)
Comment (KZavorotna) [reply src/models/environment.ts:28]: Updated

**Review by yarsky-tgz — APPROVED** [review]

**Review by everhardt — APPROVED** [review]
Looks good! Will hold off merging until platform and platform-client implementations are ready

**Pull request merged**